### ISSUE TITLE
Store component - Image block » link does't work

### ISSUE THREAD

**Issue by fpinhodev** [vtex-apps/store-discussion#310]
**Describe the bug**

_**App Store Component » Image Block**_
- The "**title**" propertie should be on "**imageElementLink**" not in "**imageElement**".
- The "**href**" propertie on "**imageElementLink**" is empty.

**Code**

![code](https://user-images.githubusercontent.com/19345522/83161730-b95a5b80-a100-11ea-8904-2320ae321135.JPG)

**Browser inspector » result**

![broweser-inspector](https://user-images.githubusercontent.com/19345522/83161811-d2fba300-a100-11ea-990f-a2f8fd5b27a7.JPG)

**Comment by victorhmp** [comment]
Hi @fpinhodev !
It appears you're trying to use the `link` prop from the `image` block with the wrong format, that's why you're seeing this weird behavior.

Here in the [docs](https://github.com/vtex-apps/store-components/blob/master/docs/Image.md) you can see that the `link` prop has a type of `Link`, which has its definition [here](https://github.com/vtex-apps/native-types/blob/f63aeeb8f6e62f4a9aaec052a8be34973be7389b/pages/contentSchemas.json#L52-L74).

Basically, what you need to do is pass an object that matches the JSON Schema defined in the second link (the one from `vtex-apps/native-types`). Such as:

```
link: {
    title: "Link title",
    url: "https://vtex.com"
}
```

Hope that helps 😄 !
